sdk/notificationhubs/notification-hubs/package.json

```diff
@@ -0,0 +1,102 @@
+    "clean": "rimraf dist dist-* temp types *.tgz *.log",
+    "execute:samples": "dev-tool samples run samples-dev",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "generate:client": "autorest --typescript ./swagger/README.md",
+    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 'test/**/*.spec.ts'",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint README.md package.json api-extractor.json src test --ext .ts,.javascript,.js --fix --fix-type [problem,suggestion]",
+    "lint": "eslint README.md package.json api-extractor.json src test --ext .ts,.javascript,.js",
+    "pack": "npm pack 2>&1",
+    "test:browser": "npm run clean && npm run build:test && npm run integration-test:browser",
+    "test:node": "npm run clean && tsc -p . && npm run integration-test:node",
+    "test": "npm run clean && tsc -p . && npm run unit-test:node && dev-tool run bundle && npm run unit-test:browser && npm run integration-test",
+    "unit-test:browser": "npm run integration-test:browser",
+    "unit-test:node": "npm run integration-test:node",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/latest/",
+    "types/3.1/",
+    "README.md",
+    "LICENSE"
+  ],
+  "repository": "github:Azure/azure-sdk-for-js",
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "keywords": [
+    "azure",
+    "cloud",
+    "typescript"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/template/template/README.md",
+  "sideEffects": false,
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "devDependencies": {
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure/test-utils": "^1.0.0",
+    "chai": "^4.3.6",
+    "downlevel-dts": "^0.10.0",
+    "eslint": "^8.16.0",
+    "@microsoft/api-extractor": "7.24.2",
+    "prettier": "^2.6.2",
+    "rimraf": "^3.0.2",
+    "@types/chai": "^4.3.1",
+    "@types/node": "^12.0.0",
+    "typescript": "^4.7.2"
+  },
+  "dependencies": {
+    "@azure/abort-controller": "^1.1.0",
+    "@azure/core-amqp": "^3.1.1",
+    "@azure/core-auth": "^1.3.2",
+    "@azure/core-client": "^1.6.0",
+    "@azure/core-paging": "^1.3.0",
+    "@azure/core-rest-pipeline": "^1.8.1",
+    "@azure/core-tracing": "^1.0.1",
+    "@azure/core-util": "^1.0.0",
+    "@azure/core-xml": "^1.2.1",
```

Comment:
are all these deps needed?

Reply:
yeah if we can ditch some that would be great, didn't see XML or AMQP used yet

Reply:
AMQP is needed for the connection string parsing.  XML will be needed when doing the registration APIs such as [Read Registrations](https://docs.microsoft.com/en-us/rest/api/notificationhubs/read-all-registrations-channel)

Reply:
Are there any features that would use AMQP link to communicate with the service?

Reply:
No, I'm purely using the package for the SAS based connection string parsing and token credentials.

Reply:
Maybe we can move those helpers out to core-util?

Reply:
We probably should.  Let's get further in the project before doing so.

Reply:
wanna file a tracking issue for it? I'm terrible at remembering this kinda stuff lately